packages/plugin-prisma/tests/__snapshots__/index.test.ts.snap

```diff
@@ -174,615 +174,6 @@ type Profile {
   user: User
 }
 
-type Query {
```

Comment:
Even if you run a snapshot update in `main` branch these changes will be present